packages/teams-js/src/public/interfaces.ts

```diff
@@ -980,3 +1008,16 @@ export enum DevicePermission {
   GeoLocation = 'geolocation',
   Media = 'media',
 }
+
+/** @hidden */
+export interface HostVersionsInfo {
+  adaptiveCardSchemaVersion?: AdaptiveCardVersion;
+}
+
+/**
+ * Represents the major and minor versions of the AdaptiveCard schema in the current host
```

Comment:
For consistency, probably should space out "Adaptive Card" (rather than PascalCase)